packages/commonwealth/server/migrations/20221101164753-discord-snapshot-updates.js

```diff
@@ -0,0 +1,219 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.createTable(
+        'DiscordBotConfig',
+        {
+          id: {
+            type: Sequelize.INTEGER,
+            autoIncrement: true,
+            primaryKey: true,
+          }, // autogenerated
+          bot_id: { type: Sequelize.STRING, allowNull: true },
+          chain_id: {
+            type: Sequelize.STRING,
+            allowNull: false,
+            references: { model: 'Chains', key: 'id' },
+          },
+          guild_id: { type: Sequelize.STRING, allowNull: true },
+          snapshot_channel_id: { type: Sequelize.STRING, allowNull: true },
+          verification_token: { type: Sequelize.STRING, allowNull: true },
+          verified: {
+            type: Sequelize.BOOLEAN,
+            allowNull: false,
+            defaultValue: false,
+          },
+          token_expiration: { type: Sequelize.DATE, allowNull: true },
+          created_at: { type: Sequelize.DATE, allowNull: false },
+          updated_at: { type: Sequelize.DATE, allowNull: false },
+        },
+        { transaction: t }
+      );
+
+      await queryInterface.addColumn(
+        'Chains',
+        'discord_config_id',
+        {
+          type: Sequelize.INTEGER,
+          references: { model: 'DiscordBotConfig', key: 'id' },
+          allowNull: true,
+        },
+        { transaction: t }
+      );
+
+      await queryInterface.addColumn('Subscriptions', 'snapshot_id', {
+        type: Sequelize.STRING,
+        allowNull: true,
+      });
+
+      await queryInterface.createTable(
+        'SnapshotProposals',
+        {
+          id: {
+            type: Sequelize.STRING,
+            allowNull: false,
+            primaryKey: true,
+          },
+          title: {
+            type: Sequelize.STRING,
+            allowNull: true,
+          },
+          body: {
+            type: Sequelize.TEXT,
+            allowNull: true,
+          },
+          choices: {
+            type: Sequelize.ARRAY(Sequelize.STRING),
+            allowNull: true,
+          },
+          space: {
+            type: Sequelize.STRING,
+            allowNull: false,
+          },
+          event: {
+            type: Sequelize.STRING,
+            allowNull: true,
+          },
+          start: {
+            type: Sequelize.STRING,
+            allowNull: true,
+          },
+          expire: {
+            type: Sequelize.STRING,
+            allowNull: true,
+          },
+          createdAt: { type: Sequelize.DATE, allowNull: false },
```

Comment:
Can we underscore these for consistency's sake and add `underscored: true` to the model?